### ISSUE TITLE
Bibliography in the end?

### ISSUE THREAD

**Issue by antagomir** [microbiome/OMA#101]
We have now bibliography at the end of each chapter. Might work better to have a single long bibliography at the end of the book. Not sure if this is easy in Rmd/Quarto, though.

**Comment by antagomir** [comment]
To be solved after #188

**Comment by ake123** [comment]
This is already done

**Comment by antagomir** [comment]
I close now. Check other issues if they can be closed, and close when done.
